diff --git a/node_modules/material-ui-dropzone/dist/index.es.js b/node_modules/material-ui-dropzone/dist/index.es.js
index 259dbd0..4243046 100644
--- a/node_modules/material-ui-dropzone/dist/index.es.js
+++ b/node_modules/material-ui-dropzone/dist/index.es.js
@@ -12,7 +12,7 @@ import PropTypes from 'prop-types';
 import { createElement, Fragment, PureComponent } from 'react';
 import Snackbar from '@material-ui/core/Snackbar';
 import Typography from '@material-ui/core/Typography';
-import { withStyles } from '@material-ui/core/styles';
+import { withStyles } from '@material-ui/styles';
 import AttachFileIcon from '@material-ui/icons/AttachFile';
 import CloudUploadIcon from '@material-ui/icons/CloudUpload';
 import clsx from 'clsx';
